### PR TITLE
Update for operate-first.cloud

### PR DESCRIPTION
**Pull request by lmzuccarelli**
- Removed all references to runAsUser (0) - root
- Update includes removal of patches (operate-frist permissions for patching is forbidden).
- Update namespace to okd-team in manifests (where applicable)
- Update Dockerfile for newer version of golang

Verification on operate-first pipelineruns

```bash

Name:              pipeline-dev-all-run-9kdmx
Namespace:         okd-team
Pipeline Ref:      pipeline-dev-all
Service Account:   pipeline
Timeout:           1h0m0s
Labels:
 tekton.dev/pipeline=pipeline-dev-all

🌡️  Status

STARTED          DURATION   STATUS
42 minutes ago   4m8s       Succeeded

⚓ Params

 NAME                    VALUE
 ∙ base-image-registry   quay.io/okd
 ∙ bundle-version        v0.0.1
 ∙ repo-name             node-observability-operator
 ∙ repo-url              https://github.com/openshift/node-observability-operator

📂 Workspaces

 NAME                 SUB PATH   WORKSPACE BINDING
 ∙ shared-workspace   ---        PersistentVolumeClaim (claimName=pvc-b739941231)

🗂  Taskruns

 NAME                                                  TASK NAME          STARTED          DURATION   STATUS
 ∙ pipeline-dev-all-run-9kdmx-bundle-all-t4qct         bundle-all         39 minutes ago   56s        Succeeded
 ∙ pipeline-dev-all-run-9kdmx-container-all-86g4s      container-all      42 minutes ago   2m45s      Succeeded
 ∙ pipeline-dev-all-run-9kdmx-fetch-repository-fbbnp   fetch-repository   42 minutes ago   27s        Succeeded


```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -111,7 +112,8 @@ kubectl create configmap docker-config --from-file=/$HOME/.docker/config.json -n
 
 ### Option 1 - On clusters with existing PVCs
 
-Execute the following to start a pipeline run
+Execute the following to start a pipeline run, this will re-use the claim "pipeline-pvc-dev" for
+future builds, it will re-use the .cache and pkd dirs to speed up builds
```
Comment (sherine-k) [inline README.md]:
```suggestion
future builds, it will re-use the .cache and pkg dirs to speed up builds
```

**Review by sherine-k — CHANGES_REQUESTED** [review]
@lmzuccarelli I spotted just a little typo in the md file.
:+1:

**Review by sherine-k — APPROVED** [review]

**Comment by lmzuccarelli** [comment]
@sherine-k - thanks for the review

**Pull request merged**